docs/products/dragonfly/howto/migrate-aiven-redis-df-console.md

```diff
@@ -1,29 +1,36 @@
 ---
 title: Migrate Aiven for Redis®* to Aiven for Dragonfly®
-limited: true
 ---
 
 Transition Aiven for Redis® databases seamlessly to Aiven for Dragonfly
-using the [Aiven Console](https://console.aiven.io/). This article
-provides detailed instructions for the migration process.
+using the [Aiven Console](https://console.aiven.io/).
 
 The Aiven Console migration tool simplifies the process of migrating
 databases to the Aiven for Dragonfly managed service.
 
+:::note
+Aiven for Dragonfly is fully supported by Aiven's service level agreements (SLAs),
+ensuring its capability to manage production workloads. As the latest addition to
+Aiven services, we recommend initiating a proof of concept (PoC)
+with Aiven for Dragonfly to thoroughly evaluate its capabilities and confirm its
+fit for your production requirements.
+:::
```

Comment:
This is repeated 3 times, so let's make it a reusable snippet to keep things clean?

1. Dump the note in `/static/includes/Dragonfly-SLA.md` or similar.
2. Import it where needed:
 
```
import Note from "@site/static/includes/Dragonfly-SLA.md"

<Note/>
```